project/AppDependencies.scala

```diff
@@ -2,31 +2,30 @@ import sbt.*
 
 object AppDependencies {
 
-  val hmrcMongoVersion = "1.7.0"
-  val bootstrapVersion = "7.23.0"
+  val hmrcMongoVersion = "1.8.0"
+  val bootstrapVersion = "8.5.0"
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-28"    % "8.5.0",
-    "uk.gov.hmrc"                  %% "play-conditional-form-mapping" % "1.13.0-play-28",
-    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28"    % bootstrapVersion,
-    "uk.gov.hmrc"                  %% "internal-auth-client-play-28"  % "1.10.0",
-    "uk.gov.hmrc.objectstore"      %% "object-store-client-play-28"   % "1.3.0",
-    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"            % hmrcMongoVersion,
-    "org.typelevel"                %% "cats-core"                     % "2.10.0",
-    "com.beachape"                 %% "enumeratum-play-json"          % "1.6.3",
-    "com.googlecode.libphonenumber" % "libphonenumber"                % "8.13.29"
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30"            % "8.5.0",
+    "uk.gov.hmrc"                  %% "play-conditional-form-mapping-play-30" % "2.0.0",
+    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-30"            % bootstrapVersion,
+    "uk.gov.hmrc"                  %% "internal-auth-client-play-30"          % "2.0.0",
+    "uk.gov.hmrc.objectstore"      %% "object-store-client-play-30"           % "1.3.0",
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-30"                    % hmrcMongoVersion,
+    "org.typelevel"                %% "cats-core"                             % "2.10.0",
+    "com.beachape"                 %% "enumeratum-play-json"                  % "1.6.3",
+    "com.googlecode.libphonenumber" % "libphonenumber"                        % "8.13.34",
+    "org.apache.pekko"             %% "pekko-connectors-csv"                  % "1.0.2"
   )
 
   val test: Seq[ModuleID]    = Seq(
-    "org.scalatest"       %% "scalatest"               % "3.2.17",
-    "org.jsoup"            % "jsoup"                   % "1.17.2",
-    "org.mockito"         %% "mockito-scala"           % "1.17.30",
-    "org.scalatestplus"   %% "scalacheck-1-17"         % "3.2.17.0",
-    "wolfendale"          %% "scalacheck-gen-regexp"   % "0.1.2",
-    "uk.gov.hmrc.mongo"   %% "hmrc-mongo-test-play-28" % hmrcMongoVersion,
-    "uk.gov.hmrc"         %% "bootstrap-test-play-28"  % bootstrapVersion,
-    "com.vladsch.flexmark" % "flexmark-all"            % "0.64.8"
+    "org.jsoup"          % "jsoup"                   % "1.17.2",
+    "org.mockito"       %% "mockito-scala"           % "1.17.31",
+    "org.scalatestplus" %% "scalacheck-1-17"         % "3.2.18.0",
+    "wolfendale"        %% "scalacheck-gen-regexp"   % "0.1.2",
```

Comment:
This version here is outdated which can be seen [here](https://github.com/wolfendale/scalacheck-gen-regexp/releases/tag/v1.1.0). However, please remove instead of updating the version. Only two files use this but don't really need the dependency to make the tests work. Use the [eoriGenerator](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/DDCE-5071/test-utils/generators/Generators.scala#L174-L180) instead. Update the lines [here](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/DDCE-5071/test-utils/generators/ApplicationRequestGenerator.scala#L26) and [here](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/DDCE-5071/test-utils/generators/TraderDetailsGenerator.scala#L31) to `eoriGenerator.map(EoriNumber)` and remove the unused imports, thanks. One less dependency to worry about :)